### PR TITLE
Added logging for debugging configs

### LINKED ISSUE #468: Display Important Settings via Log
For debugging user issues, this will help to skip a step and make the process faster for finding proper errors.

### PR DESCRIPTION
**Pull request by aidenzzz**
Fixes #468 

Let me know if anything needs to be fixed. New to the codebase :)

### CODE REVIEW

**Review by polybiusproxy — COMMENTED** [review]
I don't think it's necessary to output \*every\* setting. Preferably only the important ones such as `isNeo`, GPU (excluding screen dimensions), Vulkan, and LLE settings. I think everything else can be omitted, I believe outputting everything will pollute the log.

**Comment by aidenzzz** [comment]
Thanks. Wasn't sure which were important or not.

**Comment by lzardy** [comment]
You should consolidate this behavior to a function within `config.cpp`, such as `logConfig` function. This should recurse over config settings and provide the formatting based on the toml directly. Ex: `LOG_INFO(Config, "{}: {}", prefix, value);`

File: src/emulator.cpp [hunk 1]
```diff
@@ -52,6 +52,19 @@ Emulator::Emulator() {
...
+    LOG_INFO(Config, "GPU shouldDumpShaders: {}", Config::dumpShaders());
+    LOG_INFO(Config, "GPU shouldDumpPM4: {}", Config::dumpPM4());
+    LOG_INFO(Config, "GPU vblankDivider: {}", Config::vblankDiv());
+    LOG_INFO(Config, "Vulkan gpuId: {}", Config::getGpuId());
+    LOG_INFO(Config, "Vulkan vkValidation: {}", Config::vkValidationEnabled());
+    LOG_INFO(Config, "Vulkan vkValidationSync: {}", Config::vkValidationSyncEnabled());
+    LOG_INFO(Config, "Vulkan vkValidationGpu: {}", Config::vkValidationGpuEnabled());
+    LOG_INFO(Config, "Vulkan rdocEnable: {}", Config::isRdocEnabled());
+    LOG_INFO(Config, "Vulkan rdocMarkersEnable: {}", Config::isMarkersEnabled());
+    LOG_INFO(Config, "LLE isLibc: {}," Config::isLleLibc());
```
Comment (polybiusproxy) [inline src/emulator.cpp]:
```diff
-    LOG_INFO(Config, "LLE isLibc: {}," Config::isLleLibc());
+    LOG_INFO(Config, "LLE isLibc: {}", Config::isLleLibc());
```
Building is failing, please fix.

**Review by polybiusproxy — CHANGES_REQUESTED** [review]
Looks good to me, just needs to resolve the compilation error

**Comment by aidenzzz** [comment]
Should be fixed now. Sorry, hadn't ran it locally yet.

**Review by polybiusproxy — APPROVED** [review]

**Pull request merged**